### PR TITLE
feat: Added sticky routing support for interactive tx

### PR DESCRIPTION
**Pull request by JigarJoshi**

### CODE REVIEW

File: driver/http.go [hunk 1]
```diff
@@ -32,10 +32,13 @@ import (
 )
 
 const (
-	DefaultHTTPPort = 443
+	DefaultHTTPPort    = 443
+	SetCookieHeaderKey = "Cookie"
```
Comment (efirs) [inline driver/http.go:36]: Set-Cookie?

File: driver/grpc.go [hunk 2]
```diff
@@ -144,29 +144,43 @@ func (c *grpcDriver) dropDatabaseWithOptions(ctx context.Context, db string, opt
...
+		}
+	}
+
+	return &grpcCRUD{db: db, api: c.api, txCtx: resp.GetTxCtx(), additionalMetadata: additionalHeaders}, nil
 }
 
-func setGRPCTxCtx(ctx context.Context, txCtx *api.TransactionCtx) context.Context {
-	if txCtx == nil || txCtx.Id == "" {
+func setGRPCTxCtx(ctx context.Context, txCtx *api.TransactionCtx, additionalMetadata grpcmd.MD) context.Context {
+	if txCtx == nil || txCtx.Id == "" || additionalMetadata == nil {
```
Comment (efirs) [inline driver/grpc.go:171]: I think we don't need `|| additionalMetadata == nil` check here

File: driver/http.go (hunk 1 above)
Comment (JigarJoshi) [reply driver/http.go]: yes copy-paste typo. fixed.

File: driver/grpc.go (hunk 2 above)
Comment (JigarJoshi) [reply driver/grpc.go]: Fixed.

**Review by efirs — APPROVED** [review]

**Pull request merged**